cashflow.py

```diff
@@ -148,32 +148,30 @@ def transition_calculate(self):
         pool = Pool()
         MoveLine = pool.get('account.move.line')
         CashFlowMove = pool.get('account.cashflow.move')
-        Account = pool.get('account.account')
-        BankAccount = pool.get('bank.account')
+        BankAccountCashflow = pool.get('bank.account.cashflow')
 
         CashFlowMove.delete(CashFlowMove.search([
                     ('system_computed', '=', True),
                     ]))
 
-        bank_accounts_ids = [bank_account.account.id
-            for bank_account in BankAccount.search([('account', '!=', None)])]
-        with Transaction().set_context():
-            accounts = Account.search([('id', 'in', bank_accounts_ids)])
+        # TODO in case have many records in bank.account related at the same account.account: create cashflow same account?
+        cashflow_accounts = BankAccountCashflow.search([('company', '=', Transaction().context.get('company', -1))])
```

Comment:
I think we should keep the search using "BankAccount" because if "BankAccount" is inactive we do not want to manage it.

Reply:
See b8abb70f486d107eb699113c45ef1f1bbc8aacd3 + ecd1b18fb3278cb8bab1bdc832dd0cf73e9eaff6